app/main.py

```diff
@@ -1 +1,40 @@
-# write your code here
+def copy_file(command: str) -> None:
+    # Split the command into parts
+    parts = command.split()
+
+    # Ensure the command has 3 parts
+    if len(parts) != 3:
```

Comment:
all the checks like len of command, file_names etc can be done in one string. Also you  missing check that command = `cp` 
For these three checks no need to print anything - let your program do main logic only if these three conditions are met